backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -31,31 +40,86 @@ object BackendWorkflowInitializationActor {
 trait BackendWorkflowInitializationActor extends BackendLifecycleActor with ActorLogging {
 
   def receive: Receive = LoggingReceive {
-    case Initialize    => performActionThenRespond(initSequence, onFailure = InitializationFailed)
-    case AbortWorkflow => performActionThenRespond(abortInitialization, onFailure = BackendWorkflowAbortFailedResponse)
+    case Initialize => performActionThenRespond(initSequence(), onFailure = InitializationFailed)
+    case AbortWorkflow => performActionThenRespond(abortInitialization(), onFailure = BackendWorkflowAbortFailedResponse)
   }
 
   /**
     * Our predefined sequence to run during preStart
     */
-  final def initSequence = for {
-    _ <- validate
-    _ <- beforeAll
-  } yield InitializationSuccess
+  final def initSequence() = {
+    for {
+      _ <- validate()
+      _ <- beforeAll()
+    } yield InitializationSuccess
+  }
```

Comment:
I think @cjllanwarne meant about the extra `{` before the actual for comprehension.